bin/pom.xml

```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
we use Gradle across hmcts

Reply:
This was generated by the project when bin/code-gen.sh was executed. I think going forward I'll end up ditching the generated code and remove this.